src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -618,20 +624,48 @@ export default class ResultsViewOncoprint extends React.Component<
                     return self.horzZoom;
                 }
             },
+            get isSaveButtonEnabled() {
+                return self.props.store.pageUserSession.isDirty;
+            },
         });
-        if (!this.urlWrapper.hasClinicalTracksConfig) {
+
+        this.configureClinicalTracks();
+    }
+
+    private configureClinicalTracks() {
+        const sessionConfig = this.props.store.pageUserSession.userSettings
+            ?.clinicallist;
+        const urlConfig = this.urlWrapper.oncoprintSelectedClinicalTracks;
+
+        if (!urlConfig && !sessionConfig) {
             this.initializeClinicalTracksFromServerConfig();
+        } else if (!urlConfig && sessionConfig) {
+            // do not update deprecated clinicallist url param
+        } else if (urlConfig) {
+            this.setSessionClinicalTracks(
+                this.urlWrapper.oncoprintSelectedClinicalTracks!
+            );
         }
     }
 
     private initializeClinicalTracksFromServerConfig(): void {
```

Comment:
put a comment explaining that this applies the default set in instance configuration for clinical tracks

Reply:
Fixed in [9065f7e](https://github.com/cBioPortal/cbioportal-frontend/compare/240ae85eba446230bffd086012e309356e71f413..9065f7ea67dda257aad80c7a2b09cc88272492d9)